index.html

```diff
@@ -0,0 +1,1161 @@
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Baloo+Da+2&display=swap" rel="stylesheet">
+
+  <!-- <link href="https://fonts.googleapis.com/css?family=Barlow&#x7c;Barlow+Condensed&#x7c;Roboto+Condensed:700&#x7c;Roboto+Slab:700&#x7c;Open+Sans+Condensed:300&#x7c;=swap" rel="stylesheet"> -->
+
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" type="image/png" href="images/favicon.png"/>
+</head>
+<body class=" _robotoCond" style="background-color: rgb(244, 245, 246);">
+  <header class="mb-1 _sticky">
+    <div class="container-fluid p-0" style="max-width: 1310px;">
+      <div class="text-white d-flex align-items-end justify-content-between py-2" style="background-color: red;">
+        <div class="container p-0 pl-2  pl-lg-4 col-5 col-lg-3 align-self-center align-self-stretch d-flex">
+          <div class="row justify-content-between m-0 w-100 align-self-center">
+            <a href="#" class="col-8 col-lg-7 col-lg-6 text-left d-block d-lg-none p-0 w-100 text-decoration-none">
+              <div class="_fnt25 _robotoSlab text-white _lh1" >Newsweek</div>
+            </a>
+            <div class="col-4 col-lg-5 text-left col-lg-12 p-0 pl-0 m-0 w-100">
+              <div class="_fnt13"><a href="#" class="text-decoration-none"><img src="images/whether.png" alt=".."><span class="text-white mr-lg-2">11&deg; <span class="d-none d-lg-inline"> null, GR<img src="images/arrow-right.png" class="ml-lg-1"  alt=".."></span></span></a></div>
+              <div class="mt-2 _openSansCond font-weight-light  d-none d-lg-block"><span>Wed, Mar 18, 2020</span></div>
+            </div>
+          </div>
+        </div>
+        <a href="#" class="col-6 d-none d-lg-block display-2 _robotoSlab text-white text-center _lspc-5 text-decoration-none"><div class=" " style="line-height: 0.9em;">Newsweek</div></a>
+        <div class="col-7 col-lg-3 py-2 _fnt18 font-weight-bold _barlowCond text-right">
+          <a href="#" class="  text-white mr-2 text-decoration-none">SIGN IN</a>
+          <a href="#" class="bg-white text-danger pl-3 pr-2 py-1 text-decoration-none">SUBSCRIBE<img src="images/arrow-right-red.png" class="ml-2 mb-1" alt=".."></a>
+          <a href="" class="d-lg-none "><img src="images/menu.png" alt="..." class="mx-3"></a>
+        </div>
+      </div>
+      <!--  -->
+      <div class="header-bottom bg-white position-relative d-none d-xl-block _fnt18">
+        <img src="images/page-fold.png" alt="page-fold" class="page-fold position-absolute" style="right: 0;">
+        <nav class="navbar navbar-expand-xl d-none d-md-block">
+          <div class="collapse navbar-collapse">
```

Comment:
- [ ] Check the responsiveness of your navbar. It should show all the menus until it shows the hamburger button (around 990px) and yours is hiding it sooner.

Reply:
Corrected